docs/features.rst

```diff
@@ -114,10 +114,10 @@ Pivot Table
   exact pivot setup to share with colleagues.
 - Download the pivot view as a CSV.
 
-Displaying query results as charts
+Displaying query results as line chart
 ----------------------------------
```

Comment:
```suggestion
--------------------------------------
```
These lines need to match the length of the text.

Reply:
RST is the worst 🙄